client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -30,16 +31,11 @@ export default function Header( { isEditorSidebarOpened, toggleGeneralSidebar }:
 			aria-label={ NO__( 'Top bar' ) }
 			tabIndex={ -1 }
 		>
-			<div>
-				<p>
-					You have a: { siteType }!
-					{ siteTitle && (
-						<>
-							<br />
-							It's called <em>{ siteTitle }</em>!
-						</>
-					) }
-				</p>
+			<div className="gutenboarding__header-site">
+				{ siteType === EMPTY_FORM_VALUE && ! siteTitle && (
```

Comment:
Should only depend on `siteTitle` now?
```suggestion
				{ ! siteTitle && (
```
In that case, we can have ternary render:
```js
siteTitle ? <This /> : <That />
```